lib/chitragupta/util.js

```diff
@@ -40,6 +40,8 @@ function populateServerData(dataParam) {
 
   data.meta.format.category = categories.SERVER;
   data.meta.format.version = formatVersions.SERVER;
+
+  data.log.id = request.headers['x-chitragupta-log-id'];
```

Comment:
1. You might need to change the case of first letters.
2. We should be setting the log id from header only if not passed, right?

Reply:
Also, bump the version

Reply:
1. The Upper case gets converted to lower case by default in headers.
2. https://github.com/browserstack/chitragupta-node/pull/2/files#diff-851bb54d7ddb62d75cfccdcc586f98948eeef20a177eec727c7904553eae88faR44
3. https://github.com/browserstack/chitragupta-node/pull/2/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3-R3